netlify.toml

```diff
@@ -3,9 +3,6 @@
   targetPort = 8000
   framework = "#custom"
 
-[context.deploy-preview]
```

Comment:
Question: why did you remove this?

Reply:
Ah-ha, I meant to comment on this.

Because there is a weird caching behaviour when you add an icon, if you haven't _also_ changed something on the website, when you redeploy with without cache it won't actually happen because none of the files this is looking for has changed. It's kind of weird, because you would expect that behaviour to also happen just on a normal push, but it doesn't. So I still don't think this works in the way we want it to. So, to save confusion in the future I just removed it.